apiv6/examples/demos/retryafter_demo/main.go

```diff
@@ -0,0 +1,191 @@
+package main
```

Comment:
I suggest to adjust retry_demo so that latest retry-after functionality is showcased there to avoid excess duplications. I wonder why dupl didn't catch that.

Reply:
I wondered this too. I think `run: skip-dirs` option in linters' config file is skipping more directories than intended. I will take a look at it.

Reply:
Done